modules/http/src/test/resources/http-listener-response-streaming-config.xml

```diff
@@ -245,4 +246,21 @@
         <set-property propertyName="Content-Length" value="#[stringPayload.length()]"/>
     </flow>
 
+    <flow name="mapFlow">
+        <http:listener config-ref="listenerConfig" path="/map"/>
+        <set-property propertyName="Content-Type" value="application/x-www-form-urlencoded"/>
```

Comment:
To make the 4.x cherry-pick process easier, I'd use a response builder with the Content-Type header instead of a set-property element which will no longer work.